Examples-Web/IdleTimeout/src/Hooks/IdleTimeout.tsx

```diff
@@ -0,0 +1,33 @@
+import React, { useEffect } from 'react';
+
+import { ConnectionManager } from 'TouchFree/Connection/ConnectionManager';
+
+// Custom hook which will return true when no hands have been
+// observed for specified timeout in milliseconds.
+// ConnectionManager.init() must be called before this hook.
+const useIdleTimeout = (timeoutMs: number): boolean => {
+    const [isIdle, setIsIdle] = React.useState<boolean>(false);
```

Comment:
Could this be `true` by default? I had a second where I thought it wasn't working because it didn't have the overlay diplayed when the app loaded.